### ISSUE TITLE
Python 2.7: Cannot change kernel size in predictor convolution of SSD from 3x3 to 1x1

### ISSUE THREAD

**Issue by ndcuong91** [dmlc/gluon-cv#679]
For building mobilenetSSD300 as same as Caffe's, i need to change kernel size in predictor convolution from 3x3 to 1x1 by modify those code in gluoncv/nn/predictor.py

> def __init__(self, num_channel, kernel=(3, 3), pad=(1, 1), stride=(1, 1),
>                  activation=None, use_bias=True, **kwargs):
>         super(ConvPredictor, self).__init__(**kwargs)
>         print('predictor.py change kernel (3,3) to (1,1)')
>         kernel=(1,1)
>         pad=(0,0)
>         with self.name_scope():
>             self.predictor = nn.Conv2D(
>                 num_channel, kernel, strides=stride, padding=pad,
>                 activation=activation, use_bias=use_bias,
>                 weight_initializer=mx.init.Xavier(magnitude=2),
>                 bias_initializer='zeros') 

However, i got these error in output log:

[INFO:root:Namespace(batch_size=32, data_shape=300, dataset='voc', epochs=500, gpus='0', log_interval=10, lr=0.001, lr_decay=0.1, lr_decay_epoch='160,200', momentum=0.9, network='mobilenet1.0', num_workers=4, resume='', save_interval=5, save_prefix='ssd_300_mobilenet1.0_voc', seed=233, start_epoch=0, val_interval=1, wd=0.0005)
> INFO:root:Start training from [Epoch 0]
> python2.7: malloc.c:3722: _int_malloc: Assertion `(unsigned long) (size) >= (unsigned long) (nb)' failed.
> python2.7: malloc.c:3722: _int_malloc: Assertion `(unsigned long) (size) >= (unsigned long) (nb)' failed.](url)

When i use kernel 3x3 with pad 1x1, the training process was OK with output log like this:

[INFO:root:Start training from [Epoch 0]
[11:34:02] src/operator/nn/./cudnn/./cudnn_algoreg-inl.h:97: Running performance tests to find the best convolution algorithm, this can take a while... (setting env variable MXNET_CUDNN_AUTOTUNE_DEFAULT to 0 to disable)
INFO:root:[Epoch 0][Batch 9], Speed: 150.477 samples/sec, CrossEntropy=14.378, SmoothL1=5.289
INFO:root:[Epoch 0][Batch 19], Speed: 145.989 samples/sec, CrossEntropy=13.928, SmoothL1=5.382
INFO:root:[Epoch 0][Batch 29], Speed: 129.483 samples/sec, CrossEntropy=13.519, SmoothL1=5.341](url)

Any suggestion to fix this?
